src/YesSql.Core/Utils/HashHelper.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace YesSql.Utils
+{
+    public static class HashHelper
+    {
+        // Use of numeric characters is supported as long as the name does not start with a number.
+        private static readonly string _encode32Chars = "0123456789abcdefghjkmnpqrstvwxyz";
+
+        /// <summary>
+        /// Produces a 52 character hash.
+        /// </summary>
+        public static string HashName(string name)
+        {
+            // Create returns a fips compatabile implementation.
```

Comment:
Are there multiple issues in the sentence?

Reply:
So easiest just to remove this comment.

Here's the docs which specify fips compliance
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.sha256.create?view=net-5.0#System_Security_Cryptography_SHA256_Create